test/Microsoft.AspNet.Identity.InMemory.Test/InMemoryUserStore.cs

```diff
@@ -368,7 +368,7 @@ public Task<IList<TUser>> GetUsersInRoleAsync(string roleName, CancellationToken
         {
             if (String.IsNullOrEmpty(roleName))
             {
-                throw new ArgumentNullException("role");
+                throw new ArgumentNullException(nameof(roleName));
```

Comment:
@enghch yes and I think that was a bug! Which is part of the reason to use `nameof()` :smile: